README.md

```diff
@@ -53,6 +53,8 @@ For a more detailed overview and getting started guide, please see
      * [not_contains](#not_contains)
      * [intersects](#intersects)
      * [has_edge_degree](#has_edge_degree)
+     * [is_null](#is_null)
+     * [is_not_null](#is_null)
```

Comment:
The link on `is_not_null` is pointing to `is_null`

Reply:
A good way to self-review your README changes is to view the file itself on github, and click around, see if it renders correctly.